internal/hmyapi/backend.go

```diff
@@ -64,6 +64,9 @@ type Backend interface {
 	// Get balance
 	GetBalance(address common.Address) (*hexutil.Big, error)
 	GetShardID() uint32
+
+	// retrieve the blockHash using txID and add blockHash to CxPool for resending
+	AddBlockHashToCxPool(ctx context.Context, txID common.Hash) (uint64, bool)
```

Comment:
Name interface methods after what they achieve, not after how they are implemented:

```suggestion
	ResendCx(ctx context.Context, txID common.Hash) (uint64, bool)
```